simulator/src/main/java/com/hazelcast/simulator/coordinator/deployment/AgentWorkerLayout.java

```diff
@@ -46,27 +44,23 @@
         this.agentWorkerMode = agentWorkerMode;
     }
 
-    public List<WorkerProcessSettings> getWorkerProcessSettings() {
```

Comment:
You are looking at the wrong side or an intermediate commit. Please have a look at the final state, the methods are not public anymore.